pkg/network/openshift_sdn.go

```diff
@@ -143,7 +143,7 @@ func fillOpenShiftSDNDefaults(conf, previous *netv1.NetworkConfigSpec, hostMTU i
 	// However, this can never change, so we always prefer previous.
 	if sc.MTU == nil {
 		var mtu uint32 = uint32(hostMTU) - 50 // 50 byte VXLAN header
-		if previous != nil {
+		if previous != nil && previous.DefaultNetwork.OpenShiftSDNConfig != nil && previous.DefaultNetwork.OpenShiftSDNConfig.MTU != nil {
```

Comment:
this is an unrelated fix that should get its own commit with its own commit message

Reply:
bah, fine.